### ISSUE TITLE
(ISO 8601-1/Amd 1) Definition list within a list loses its indents

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#677]
Left: ISO/CS. Right: Metanorma.

Two occurrences:
<img width="1920" alt="Screenshot 2022-05-17 at 11 39 15 AM" src="https://user-images.githubusercontent.com/11865/168723785-bb1a4172-a3cb-48b8-bf06-8a76f274afc5.png">

This is in Metanorma.
<img width="958" alt="Screenshot 2022-05-17 at 11 40 32 AM" src="https://user-images.githubusercontent.com/11865/168723880-e920455e-fbbb-4657-a726-5dbe49a649f5.png">

**Comment by opoudjis** [comment]
The indent of 1cm as a default on dl tables was implemented for ISO, and is replicated in DIS. It words well for tables inside lists inside blockquotes (which have a div wrapper for list continuation), but for the latter instance (since we have dropped the div wrapper), it is the difference between 0.75cm and 1.0cm left margin, which isn't impressive, but isn't necessarily wrong either. Going with that for now.
